lib/radiator/directory.ex

```diff
@@ -1,14 +1,13 @@
 defmodule Radiator.Directory do
   @moduledoc """
-  The Directory context.
+  The Directory context supplying published information.
   """
 
   import Ecto.Query, warn: false
 
-  alias Radiator.Directory.Episode
-  alias Radiator.Directory.Podcast
-  alias Radiator.Directory.Network
+  alias __MODULE__
   alias Radiator.Repo
+  alias Directory.{Network, Episode, Podcast}
```

Comment:
General style question here: should aliases already include previous aliases? I tend to like it and have some changes here. E.g.

```elixir
alias Radiator.Directory
alias Directory.Episode
```
vs
```elixir
alias Radiator.Directory
alias Radiator.Directory.Episode
```

Reply:
Okay, then I'll stop this for now. My dislike for redundancy produced this.